### ISSUE TITLE
various Tensorflow versions used

### ISSUE THREAD

**Issue by KiraHune** [7fantasysz/MSCRED#2]
In the MSCRED_TF file various Tensorflow versions are used. 
It is not possible to get the program worked.

With the use of tensorflow v1 the placeholder attribute is available, but tf.contrib can not be used.
In Tensorflow 2 the placeholder attribute is not supported.

Does anyone solved this problem?

**Comment by xuesongwang** [comment]
Hi,
I used tf 1.6.0 and it works fine with me. Which exact line in the code that you got the error?

**Comment by KiraHune** [comment]
Hello,
thank you for your answer. With tf 1.6.0 the old Problems are solved, thank you.
 
But in the line 359 (saver = tf.train.Saver(max_to_keep = 10) I got a new error: tensorflow.python.framework.errors_impl.NotFoundError: Unsuccessful TensorSliceReader constructor: Failed to find any matching files for ../MSCRED/49.ckpt
I think the error is caused by the replacement of the path for the model, the raw_data and the Matrix_data in the lines 33 until 38. I could not find the file with the names in your code in the git repository.
I replaced the Arguments for the raw_data_path with the path to the file „synthetic_data_with_anomaly-s-1.csv“ and the Matrix_data_path with the path to the matrix_data, but I am not sure whether this was right.
 
But don't inconvenience yourself. I found another implementation of the MSCRED Framework on GitHub which works for my purposes, too.
 
Best regards,
Kira Huneke
Von: xuesongwang
Gesendet: Donnerstag, 17. Oktober 2019 02:49
An: KONI-SZ/MSCRED
Cc: KiraHune; Author
Betreff: Re: [KONI-SZ/MSCRED] various Tensorflow versions used (#2)

Hi,
I used tf 1.6.0 and it works fine with me. Which exact line in the code that you got the error?
